MISSING_SEQUENCE_DETECTION.md

```diff
@@ -0,0 +1,117 @@
+# Missing interchange sequence detection
+
+In rare occasion of an in-code exception there is a chance, 
+that a sequence number will be generated and registered in NHAIS database as used
+but due to the error no message will be sent with that number.
+
+Such situation will produce gaps in sequence numbers sent to HA
+
+This document describes how to produce a report on missing sequence numbers.
+
+## Database
+
+NHAIS is using MongoDb as persistence store. 
```

Comment:
This adaptor uses a persistence store (supports either DynamoDB or MongoDB) configurable based on your own implementation/hosting preferences.

Reply:
@MattLewis80 NHAIS is written only with MondoDB support

Reply:
yes but the idea behind the use of Mongo is that its compatible with other proprietary nosql db types allowing suppliers to choose their own.  Can you therefore change the text so its agnostic:
'This adaptor uses a persistence store (supports either DocumentDB for AWS or CosmoDB for Azure) configurable based on your own implementation/hosting preferences.'

Reply:
'This adaptor uses a persistence store (supports any MongoDB compatible database) configurable based on your own implementation/hosting preferences.'